src/autopas/containers/verletClusterLists/traversals/VerletClustersColoringTraversal.h

```diff
@@ -30,6 +28,8 @@ class VerletClustersColoringTraversal : public CBasedTraversal<ParticleCell, Pai
                                         public VerletClustersTraversalInterface<typename ParticleCell::ParticleType> {
  private:
   using Particle = typename ParticleCell::ParticleType;
+  typedef typename VerletClusterMaths::index_t index_t;
```

Comment:
I had this like you suggested, but the compiler seems to do something strange here, as he gives the following warning:
`warning: declaration 'using index_t = std::size_t' does not declare anything`

I would just leave it like it is with typedef and without the warning, since the index_t type is removed in my refactoring in #319, anyway. (I hope that that is okay, by the way, but I didn't see the point, especially not when using auto.)

Reply:
when it is removed anyways it is ok.